README.md

```diff
@@ -12,6 +12,22 @@ API](https://developers.google.com/google-ads/api/docs/start).
 
 ## Requirements
 
+*   Two PHP extensions need to be installed to use the library at runtime
+    *   gRPC: follow the section `Install the gRPC PHP extension`
+    of [this page](https://grpc.io/docs/quickstart/php.html)
```

Comment:
Hanging indent should be done here too. In other words, please align "of" with "gRPC" in the previous line.

Reply:
Done